gems/aws-sdk-core/spec/aws/waiters_spec.rb

```diff
@@ -190,14 +189,60 @@ module Waiters
             expect(attempts).to be(1)
           end
 
-          it 'fails when an expected error is not encountered' do
+          it 'fails when an expected status is not encountered' do
             expect {
               client.wait_until(:table_not_exists, table_name:'table') do |w|
                 w.delay = 0
               end
             }.to raise_error(Errors::TooManyAttemptsError)
           end
+        end
+
+        describe 'error matcher' do
```

Comment:
Do we have test cases for all of the possible options? This fixture seems pretty lightweight. Let's talk offline but I think we should have a test suite of all the possible options (and remove the ddb fixture).